locales/en.yml

```diff
@@ -153,3 +153,6 @@ en:
         status:
           running: running
           stopped: stopped
+        unsupported_service: |-
+          Sorry, service '%{service}' is not supported.
+          Supported services are docker, openshift and kubernetes.
```

Comment:
Should this be a variable and not a static entry to support better updates when more services are added?